packages/wdio-cli/src/run.js

```diff
@@ -1,11 +1,32 @@
+import fs from 'fs'
+import path from 'path'
 import logger from 'wdio-logger'
 
 import Launcher from './launcher.js'
+import setup from './setup'
 
 const log = logger('wdio-cli:run')
 
-export default function run (wdioConf, params) {
+export default function run (params) {
     let stdinData = ''
+    const commands = fs.readdirSync(path.join(__dirname, 'commands')).map((file) => file.slice(0, -3))
+    const localConf = path.join(process.cwd(), 'wdio.conf.js')
+    const wdioConf = params._[0] || (fs.existsSync(localConf) ? localConf : null)
```

Comment:
`params._[0]` is used twice in the `run` function. I think it's better to extract it to constant with more meaningful name.